core/app/models/spree/address.rb

```diff
@@ -28,10 +28,6 @@ class Address < Spree::Base
 
     self.allowed_ransackable_attributes = %w[name]
 
-    scope :with_values, ->(attributes) do
-      where(value_attributes(attributes))
-    end
-
```

Comment:
Is it available in all versions of Rails we currently support?

Reply:
No, it's introduced in 7.1 (I think). Probably needs a version check. It's breaking with 7.1 though with a scope already defined error.